core/src/com/unciv/ui/worldscreen/unit/UnitActions.kt

```diff
@@ -284,9 +284,10 @@ object UnitActions {
                     tile.improvement = null
                     if (tile.resource != null) tile.getOwner()?.updateDetailedCivResources()    // this might take away a resource
 
-                    val freePillage = unit.hasUnique("No movement cost to pillage")
-                        || unit.civInfo.hasUnique("No movement cost to pillage")
-                        || (unit.baseUnit.isMelee() && unit.civInfo.hasUnique("Melee units pay no movement cost to pillage"))
+                    val freePillage = unit.hasUnique(UniqueType.NoMovementToPillage)
+                        || unit.civInfo.hasUnique(UniqueType.NoMovementToPillage)
```

Comment:
Shouldn't these be combined into `unit.hasUnique(NoMovementToPilalge, checkCivUniques=true)`?

Reply:
Perhaps, that's a change I was thinking of as well
So far those seems to be the only usage it would have though, but if I see another place I'll make the change